Hai Reveal: Jump Drive plot hole & text walls in B09-B12 & continuity problems & B08 log glitch (#7677)
endless-sky/endless-sky: data/hai/hai reveal 5 philosophers.txt

		conversation
			`Despite expectations, you are told to leave Giti on the ship, and a man who immediately identifies himself to you as Deputy Director Stephen Moray meets you at the entrance of the docking tube at the spaceport and gestures you into the private antechamber attached to each dock in the <origin> port. "I work for Syndicate Internal Affairs. We're the internal police of the Syndicate; when an employee of any rank does misdeeds, we track them down. Janitor or executive; it doesn't matter. My department of Internal Affairs works on piracy-related matters.`
			`	After you both sit, Stephen Moray turns on a device and places it on the table. "Communications jammer," he explains, "so we're not heard.`
			choice
				`	"What is this about?"`
					goto about
				`	"I don't appreciate how your goons treated me."`
			`	He shrugs. "Those 'goons' were from Republic Intelligence. I have no say as to how they do their jobs. Now that you're here, let's get down to business.`
			label about

> 			`	"I know you've been working closely with the Hai lately, helping to achieve lasting stability. I communicated with Remington and he's informed me that while it might that you've caused a mess, the Hai representatives explained they were stalling while you find some information. It would seem that, as usual, there much going on behind the scenes and you have become embroiled in it.`

Reviewer: ```suggestion
			`	"I know you've been working closely with the Hai lately, helping to achieve lasting stability. I communicated with Remington and he's informed me that while it might look like you've caused a mess, the Hai representatives explained they were stalling while you find some information. It would seem that, as usual, there much going on behind the scenes and you have become embroiled in it.`
```

> 			`	"I know you've been working closely with the Hai lately, helping to achieve lasting stability. I communicated with Remington and he's informed me that while it might look like you've caused a mess, the Hai representatives explained they were stalling while you find some information. It would seem that, as usual, there much going on behind the scenes and you have become embroiled in it.`

Reviewer 2: This needs to be either "achieve an outcome with" lasting stability or "to contain the impacts of this crisis".
The language isn't appropriate for this character cut down to what it currently is.

Author: I restored the original paragraph.

			`	"I also understand what the Hai are grappling with right now," he says. "When I was a teenager, pirates kidnapped my brother and father, and forced them to serve on pirate vessels, calling them 'indentured servants.' My father watched helplessly as a Navy Frigate disabled my brother's just as it exited hyperspace. He escaped - but he's now serving his seventeenth year of a twenty-five-year sentence for piracy, even though his deeds were against his will," Moray explains as he lays out reports to emphasize his point.`

> 			`	"I tell you this because it is important you understand I'm on your side. We haven't met before, but we must trust one another. You see, the hails of the Navy: loyalty, honor, bravery - half-truths at best. We see the Navy as heroes because they are the only large organization capable of self-sacrifice. Yes, they do many good deeds, but those deeds blind us to their brutality and immorality. Not all humans are like that - many members of the Navy disagree with the barbaric rules about slavery. Some of us in the Syndicate, Free Worlds, and elsewhere, are trying to solve this problem.`

Reviewer: ```suggestion
			`	"I tell you this because it is important you understand I'm on your side. We haven't met before, but we must trust one another. You know the hails of the Navy: loyalty, honor, bravery - half-truths at best. We see the Navy as heroes because they are the only large organization capable of self-sacrifice. Yes, they do many good deeds, but those deeds blind us to their brutality and immorality. Not all humans are like that, many members of the Navy disagree with the barbaric rules about slavery. Some of us in the Syndicate, Free Worlds, and elsewhere, are trying to solve this problem.`
```

> 			`	"In my time as a Syndicate executive, I have arranged for over eighty thousand slaves to be smuggled away from pirate planets to gain their freedom in Syndicate space. I have extensive contacts within the slaver community."`

Reviewer 2: This is an oversight from the original text, but as a *deputy* director he's not an executive.
```suggestion
			`	"In my time here, I have arranged for over eighty thousand slaves to be smuggled away from pirate planets to gain their freedom in Syndicate space. I have extensive contacts within the slaver community."`
```

			`	It would seem that this is the opportunity to hasten a desirable outcome most swiftly, though maybe the Syndicate doesn't need to know about the Hai's theories of cadmium poisoning from alien technology being used.`
			choice
				`	"Here's some case location data on kidnapping victims." (Share only case location data, and don't mention the cadmium poisoning.)`
					goto mission
				`	"I have a lot of data I'd like you to analyze." (Don't mention the cadmium poisoning, but share all other data.)`
			action
				set "syndicate learns of alien tech anyway"
			`	Director Moray looks through some of the data you share while you summarize the key points. "Thank you very much, we will have our experts go over this promptly."`
			label mission
			`	With your explanation finished, he seems to straighten in preparation for something. "I have some leads to follow up. Someone has an extensive array of contacts, possibly close to home in the Syndicate - perhaps even executives, but for certain at least a department head somewhere. This is too much to be entirely independent. I can say for sure though that they're doing this without permission from the Syndicate proper.`

> 			`	"The Hai have insisted they be involved in any solution involving captured Hai. Which means I will have to send you back to them soon, but before that I need information. So do you, but my due diligence demands I ensure you are properly equipped to retrieve what I need. Take Giti and the scanning equipment I am now loading onto your ship. I recommend a ramscoop for safety. Investigate the systems and bring her back here, after that you'll have to go back to the Hai without her."`

Reviewer 2: ```suggestion
			`	"The Hai have insisted they be involved in any solution which could involve captured Hai. Which means I will have to send you back to them soon, but before that I need information. So do you, but my due diligence demands I ensure you are properly equipped to retrieve what I need. Take Giti and the scanning equipment I am now loading onto your ship. I recommend a ramscoop for safety. Investigate the systems and bring her back here, after that you'll have to go back to the Hai without her."`
```

		log `Found a bizarre planet with a subterranean colony in its moon. The planet defies analysis and seems to violate the known laws of physics. There is enough evidence of Hai tech to assume Hai presence in addition to the humans able to be detected. However, the scanners could not see the full extent of what's there due to interference from the planet.`
		conversation
			`Giti turns on the scanning equipment.`
			label scan
			choice
				`	"Tell me about the moon."`
					goto moon
					to display
						not "Hai Reveal [B09] Scanning Devil-Hide: label moon"
				`	"Tell me about that planet."`
					goto planet
					to display
						not "Hai Reveal [B09] Scanning Devil-Hide: label planet"
				`	"We found what we need. Let's get out before we're noticed."`
					goto leave
					to display
						has "Hai Reveal [B09] Scanning Devil-Hide: label moon"
			label planet
			action
				set "Hai Reveal [B09] Scanning Devil-Hide: label planet"
			`	"Okay, the planet..." Her voice trails off. "Wow! That's odd. This scanner is definitely not designed to properly scan this thing; I can only get surface readings."`

> 			`	"Whatever it is, this planet is not normal by any stretch. It is completely inhospitable and appears to have a huge network of caves and caverns with crystalline structures that are radiating... something. Whatever it is, it's messing with the scanner's attempts to identify it. I'd have to know more about this scanner's functions to guess what the interference is. There's deuterium though maybe denizens of this system siphon the deuterium to make hyperspace fuel? Doesn't seem very efficient, but the solar wind is thin here. There's some kind of gravitational anomaly though, or maybe its just some kind of interference lensing. It looks like the planet has about a thousand times the mass it should, but with gravity around Earth normal. That... can't be right."`

Reviewer: ```suggestion
			`	"Whatever it is, this planet is not normal by any stretch. It is completely inhospitable and appears to have a huge network of caves and caverns with crystalline structures that are radiating... something. Whatever it is, it's messing with the scanner's attempts to identify it. I'd have to know more about this scanner's functions to guess what the interference is. There's deuterium, though. Maybe the denizens of this system siphon the deuterium to make hyperspace fuel? Doesn't seem very efficient, but the solar wind is thin here. There's some kind of gravitational anomaly though, or maybe its just some kind of interference lensing. It looks like the planet has about a thousand times the mass it should, but with gravity around Earth normal. That... can't be right."`
```

> 			`	She shakes it off and looks at you. "Whatever it is we're looking for its not down there."`

Reviewer: ```suggestion
			`	She shakes it off and looks at you. "Whatever it is we're looking for, it's not down there."`
```

				goto scan
			label moon
			action
				set "Hai Reveal [B09] Scanning Devil-Hide: label moon"

> 			`	The moon... hmmm... okay. It's just an ordinary chunk of rock, but radiation from the planet is disturbing the sensors. I can only see a few dozen meters below ground even though the scanner should be able to look down at least ten times that far... Ahah!"`

Reviewer: ```suggestion
			`	The moon? Hmmm... okay, it's just an ordinary chunk of rock, but radiation from the planet is disturbing the sensors. I can only see a few dozen meters below ground, even though the scanner should be able to look down at least ten times that far... Aha!"`
```

> 			`	"I found something. There's a facility buried in the moon's surface. There is definitely a lot of Hai tech too, so looks like the concerns you told me about were spot on. I can only see the uppermost part of the facility but it appears to be divided into two parts. One section has hydroponics, big cargo bays, docking bays for ships, even a small shipyard. Doesn't look like they have space for more than a handful of large ships though. It also seems to have the only decently sized air processor I can identify. The other section is deeper, and appears to have spacious living areas I can see some support systems, but it seems that the air supply is effectively controlled from the surface access section. I can detect 1,387 people, though no details about them, and that's just in the upper three floors that I can see."`

Reviewer: ```suggestion
			`	"I found something. There's a facility buried in the moon's surface. There is definitely a lot of Hai tech too, your concerns were spot on. I can only see the uppermost part of the facility, but it appears to be divided into two parts. One section has hydroponics, big cargo bays, docking bays for ships, even a small shipyard. Doesn't look like they have space for more than a handful of large ships, though. It also seems to have the only decently sized air processor I can identify. The other section is deeper, and appears to have some spacious living areas. I can see some support systems, but it seems that the air supply is effectively controlled from the surface access section. I can detect 1,387 people, though no details about them, and that's just in the upper three floors that I can see."`
```

			`	She looks at you eyes wide. "This isn't just a base; there's a base down there but it's controlling a whole colony."`

> 			`	There is a pregnant pause for a moment and then she suddenly gasps. "Captain, this must be Elenchus!"`

Reviewer: ```suggestion
			`	There is a meaningful pause for a moment, and then she suddenly gasps. "Captain, this must be Elenchus!"`
```

> 			`	She looks at you eyes wide. "This isn't just a base; there's a base down there but it's controlling a whole colony."`
> 			`	There is a pregnant pause for a moment and then she suddenly gasps. "Captain, this must be Elenchus!"`

Reviewer: ```suggestion
			`	She looks at you, eyes wide. "This isn't just a base; there's a base down there, but it's controlling a whole colony."`
			`	There is a meaningful pause for a moment and then she suddenly gasps. "Captain, this must be Elenchus!"`
```

Author: The word "meaningful" does not have the same meaning as "pregnant" (even in the non-reproductive sense of the word). It is not a suitable replacement.

Reviewer: 'Meaningful' means both long and importance here, that was what pregnant meant here. It was the closest synonym I could find.

Author: It doesn't express the type of meaning, though. Why do we need to give the pause an abortion? Isn't "pregnant" a sufficient adjective?

I put in your two commas, though, in another commit just now.

				goto scan
			label leave

> 			`	As you say this, you notice on your regular sensors that there are some ships heading your way and, more horrifyingly, some ground-based turrets coming online.`

Reviewer 2: The player does need to be told they should leave, unless we'd like to add a system hazard to the planet that the player is unlikely to be able to tank the damage from.

Author: I put back in "It's time to go."

		has "Hai Reveal [B10] Report to Teeneep: done"
	on offer
		conversation
			`When you return to <origin> to tell Teeneep what you have found, she seems to already know much of what you have to say. "We've been waiting for you. Thanks to your visit to the Syndicate, we've found a lot of information in very short order. You'll have to lead another international task force through the wormhole at Devil-Eye. This time, they'll have Jump Drives from the Unfettered, which the Hai are renting for quite a hefty price. We expect all Jump Drives to be returned intact. Everyone is gathering to brief you on the status so you can prepare for the mission. A high-ranking Marshal Kaine is here with more information."`

> 			`	Sayari seems anxious when she joins you on the way, and the two of them draw you along towards one of the meeting areas in the secured spaceport. There you are led past a selection of Navy personnel on guard into a meeting room with several people, including yourself. Some appear to have just arrived but it appears that some had been here for a while, and to your surprise Alondo and Remington appear to have come in person. The place looks to have been acting as a war room of sorts. As soon as you're in someone says, "We're all here," and Choot'k and Remington stand up from where they'd been looking at something intently.`

Reviewer: ```suggestion
			`	Sayari seems anxious when she joins you on the way, and the two of them draw you along towards one of the meeting areas in the secured spaceport. There you are led past a selection of Navy personnel on guard into a meeting room with several people. Some appear to have just arrived, but others must have been here for a while, and to your surprise Alondo and Remington are also present. The place looks to have been acting as a war room of sorts. As soon you've walked through the door, someone says, "We're all here," and Choot'k and Remington stand up from where they'd been looking at something intently.`
```

					goto smuggling
					to display
						not "Hai Reveal [B11] Moving Together: label smuggling"
				`	"How have you dealt with the smugglers so far?"`
					goto dealt
					to display
						has "Hai Reveal [B11] Moving Together: label smuggling"
						not "Hai Reveal [B11] Moving Together: label dealt"
				`	"How do the Hai feel about the approach to the smugglers?"`
					goto hai
					to display
						has "Hai Reveal [B11] Moving Together: label smuggling"
						not "Hai Reveal [B11] Moving Together: label hai"
				`	"Are you still able to keep the Hai a secret?"`
					goto secret
					to display
						not "Hai Reveal [B11] Moving Together: label secret"
				`	"What do you know about Elenchus?"`
					goto elenchus
					to display
						not "Hai Reveal [B11] Moving Together: label elenchus"
				`	"How will we penetrate the Elenchus defenses?"`
					goto penetration
					to display
						has "Hai Reveal [B11] Moving Together: label elenchus"
						not "Hai Reveal [B11] Moving Together: label penetration"
				`	"We've discussed enough. It's time to get started."`
					goto solution
			label smuggling
			action
				set "Hai Reveal [B11] Moving Together: label smuggling"

> 			`	Remington looks at you. "I know its rough when news travels ahead of you, but Moray's team identified two bases which act as holding facilities in human space for the previously unidentified group that you've been investigating. That's where it seems the largest number of kidnapped Hai have been taken for processing before being taken to this mysterious 'Elenchus' place you've found. One of the bases is on Deadman's Cove, and arrangements are already being made to deal with it, but the other one is right here on Stormhold. In fact it narrowed down our remaining search here to a single target in one area up in the north. It should take us no more than a few days to pinpoint it, and a few days more to plan and execute an action against it." He shrugs. "The information was just too important to delay it for your arrival. That, and they decided they wanted me here personally since Terry appears to be busy off doing something else."`

Reviewer: ```suggestion
			`	Remington looks at you. "I know it's rough when news travels ahead of you, but Moray's team identified two bases which act as holding facilities in human space for the previously unidentified group that you've been investigating. That's where the largest number of kidnapped Hai have been taken for processing, before being taken to this mysterious 'Elenchus' place you've found. One of the bases is on Deadman's Cove, and arrangements are already being made to deal with it, but the other one is right here on Stormhold. In fact, it narrowed down our remaining search here to a single target in one area up in the north. It should take us no more than a few days to pinpoint it, and a few days more to plan and execute an action against it." He shrugs. "The information was just too important to delay it for your arrival. That, and they decided they wanted me here personally since Terry appears to be busy off doing something else."`
```

				goto briefing
			label dealt
			action
				set "Hai Reveal [B11] Moving Together: label dealt"

> 			`	Marshal Khaine continues, "While there is intel on some captured Hai in the North, we have worked out a program to address that side of things more subtly and with a lower risk to all parties involved. Raiding pirate ships is a deadly affair and their bases more so. While it may seem heroic, even with the help of the Hai the casualty rate is unsustainable. However, while they may be allowed to go by intact, we have much better tabs on slaver and smuggler ships in the north, so we have options that don't require the sort of thorough manhunt that's occurring here to be effective."`

Reviewer: ```suggestion
			`	Marshal Khaine continues, "While there is intel on some captured Hai in the North, we have worked out a program to address that side of things more subtly, and with a lower risk to all parties involved. Raiding pirate ships is a deadly affair and their bases more so. While it may seem heroic, even with the help of the Hai, the casualty rate is unsustainable. However, while they may be allowed to go by intact, we have much better tabs on slaver and smuggler ships in the north, so we have options that don't require the sort of thorough manhunt that's occurring here to be effective."`
```

				goto briefing
			label hai
			action
				set "Hai Reveal [B11] Moving Together: label hai"
			`	Sayari speaks up. "The Elders have accepted the proposed plans, but with this new information we have decided that this organization you've uncovered must still be dealt with directly. It is our main concern now. As long as the Republic remains willing to negotiate on how they deal with smugglers in the future, resolving this one issue will be sufficient for us to move forward. The overwhelming majority of the other groups who appear to have been exploiting Hai seem to have been here on <origin> and so we consider there to have been serious inroads made into resolving the problem more broadly."`
				goto briefing
			label introductions
			action
				set "Hai Reveal [B11] Moving Together: label introductions"
			`	Remington responds. "Good point. I don't think everyone here knows everyone yet."`

> 			`	You look around the room as Remington does introductions for the benefit of the new people and count off Teeneep, Sayari, Choot'k, Batu, Danforth, Adurath Khaine a Marshal for the Navy - which is an ex-service political appointment, Alondo, and Remington himself.`

Reviewer: ```suggestion
			`	You look around the room as Remington does introductions for the benefit of the new people, and counts off Teeneep, Sayari, Choot'k, Batu, Danforth, Adurath Khaine - a Marshal for the Navy, which is an ex-service political appointment - Alondo, and himself.`
```

> 			`	Remington clearly did not need to be introduced, but doing so gives him a disarming charm and you see for a moment some of the charisma that he must utilize as head of PR.`

Reviewer: ```suggestion
			`	Remington clearly did not need to be introduced, but doing so gives him a disarming charm, and you see for a moment some of the charisma that he must utilize as head of PR.`
```

> 			`	Marshal Kaine introduces himself to you directly. "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met my admiral Danforth, and I'm here because occasionally the Navy needs a politician, and the politicians need someone from the service that understands politicians."`

Reviewer: ```suggestion
			`	Marshal Kaine introduces himself to you directly. "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've worked with my admiral William Danforth in the past, and he has spoken highly of your work. I'm here because occasionally the Navy needs a politician, and the politicians need someone from the service that understands politicians."`
```
I wonder if this bit could be changed if you made everyone unhappy at the treaty negotiations.

Author: Not yet, because that's in another PR. It would have to be done after #7594 is merged.

> 			`	Batu jumps in. "The Marshal has been very effective in helping to develop some solutions with Hiyamaharu and has just arrived here to make sure everything is set in order."`

Reviewer: ```suggestion
			`	Batu jumps in. "The Marshal has been very effective in helping to develop some solutions with Hiyamaharu, and has just arrived here to make sure everything is set in order."`
```

				goto briefing
			label secret
			action
				set "Hai Reveal [B11] Moving Together: label secret"
			`	Remington explains. "This operation has gone on long enough that the popular opinion is that the Hai must obviously exist, even though there are no explicit voices saying as such. It has in fact almost become a joke; the perception being that, 'The Navy are blockading the Hai to hide humanity from them, because the blockade isn't doing a good job of being inconspicuous.' There have been memes of the Navy hiding their ships behind far too narrow light poles while they distract a squirrel with their headlights. Now it's a matter of being able to present good news. Place a successful spin on things."`
			`	"Our problem," says Khaine. "Is Elenchus."`
				goto briefing
			label elenchus
			action
				set "Hai Reveal [B11] Moving Together: label elenchus"

> 			`	Admiral Kaine explains. "Reaching Elenchus would seem to be a problem. The Unfettered are willing to loan us a few jump drives. Additionally, the base on Elenchus appears to be a hardened target. It will need specialized and suitably equipped shock troops to be sure of taking it. In our assessment so far we have determined that the Syndicate have suitable troopers they can mobilize quickly, but no way of getting them there, and our suitable shock troops are largely pre-occupied either being needed here on <origin>, or are at least a week away from being available. We need a resolution sooner than that."`

Reviewer: ```suggestion
			`	Marshal Kaine explains, "Reaching Elenchus would seem to be a problem, because it is behind systems we cannot reach with our hyperdrives. However, the Unfettered are willing to loan us a few jump drives. Additionally, the base on Elenchus appears to be a hardened target. It will need specialized and suitably equipped shock troops to be sure of taking it. In our assessment so far we have determined that the Syndicate have suitable troopers they can mobilize quickly, but no way of getting them there, and our suitable shock troops are largely pre-occupied; either being needed here on <origin>, or are at least a week away from being available. We need a resolution sooner than that."`
```

			choice
				`	"Is it really so urgent?"`

> 				`	"What's the rush?"`

Reviewer: I'm sure the player should be less politically blind than this.

Author: I just updated these options so the player can be less blind and skip the explanation.

> 			`	Khaine sighs and responds. "The thing about governments is that all crises are either to be solved, delayed, or weathered. This crisis cannot be delayed, and so they must choose to weather or resolve. There are enough representatives who feel that their positions are too fragile to weather it, and so they have pushed to resolve. The timeline is set by their perception of the urgency." He looks to the Hai. "And obviously, there are also those who value our historic relationship with the Hai and seek to resolve it for those reasons too. Those are only the first two reasons on a long list. At the end of the day, it's politics."`

Reviewer: ```suggestion
			`	Khaine sighs, as if he expected something else of you, and responds. "The thing about governments is that all crises are either to be solved, delayed, or weathered. This crisis cannot be delayed, and so they must choose to weather or resolve. There are enough representatives who feel that their positions are too fragile to weather it, and so they have pushed to resolve. The timeline is set by their perception of the urgency." He looks to the Hai. "And obviously, there are also those who value our historic relationship with the Hai and seek to resolve it for those reasons too. Those are only the first two reasons on a long list. At the end of the day, it's politics."`
```

				goto briefing

			label penetration
			action
				set "Hai Reveal [B11] Moving Together: label penetration"

> 			`	Teeneep explains. "We also need a way to deal with the surface turrets and, short of bringing overwhelming firepower which we do not have, there's only one ship available with the systems to be able to affect that within our timeline. That ship is my Centipede. I have installed electronic warfare systems on it to baffle pirate defenses when I was investigating this before. It is our assessment that I should be able to baffle the turrets long enough to deal with any spaceship defenders and land where the turrets will no longer matter." She looks to Choot'k. "The General has generously offered to loan a drive for my use."`

Reviewer: ```suggestion
			`	Teeneep explains, "We also need a way to deal with the surface turrets and, short of bringing overwhelming firepower which we do not have, there's only one ship available with the systems to be able to affect that within our timeline. That ship is my Centipede. I have installed electronic warfare systems on it to baffle pirate defenses when I was investigating this before. It is our assessment that I should be able to baffle the turrets long enough to deal with any spaceship defenders, and land where the turrets will no longer matter." She looks to Choot'k. "The General has generously offered to loan a drive for my use."`
```
Where or when (the turrets will no longer matter)?

> 			`	Choot'k addresses the group. "The designs of the facility show a weak point in one of the hangars that we specifically should be able to exploit. My flagship has weapons pillaged from another species, which are strong enough to cut open the hangar doors yet precise enough not to destroy the hangar.`

Reviewer: ```suggestion
			`	Choot'k addresses the group. "The designs of the facility show a weak point in one of the hangars, which we can exploit. My flagship has weapons pillaged from another species, which are strong enough to cut open the hangar doors yet precise enough not to destroy the hangar.`
```

> 			`	"I also have two trusted captains who have been exploring the value of human weapons while they've been here, and think they have some winning combinations they would like to test in battle. I have chosen them to make use of our remaining available drives, and out of consideration for you humans. It would seem that the fewer alien weapons we carry through your space the better."`

Reviewer: ```suggestion
			`	"I also have two trusted captains who have been exploring the value of human weapons while they've been here, and think they have some winning combinations they would like to test in battle. I have chosen them to make use of our remaining available drives, and out of consideration for you humans. It would seem that the fewer alien weapons we carry through your space, the better."`
```

					goto control
				`	"That equipment sounds ominous."`
			`	Kopkow gives you a disturbing smile that has an 'I love my work' appearance. He departs as everyone else assembles.`
				goto leave
			label control
			`	"It's more of a collaboration, really, but like I said: need-to-know basis." He nods to you and departs as everyone else assembles.`
			label leave
			`	After a large meeting about logistics, and an unsurmountable buffet, everyone disperses. General Choot'k finds you to discuss critical tactical matters.`
			`	"You have handled yourself well throughout this. I want to see you prove yourself in battle before I open the entrance to <planet>. When we arrive at <planet>, we'll first fight the defense fleet waiting for us. I suspect we can handle it, but you are of course welcome to assist.`
			`	"The main priority for you should be to make sure nobody pursues Elder Teeneep or Captain Wallace. However, these scum will likely keep a couple of ships in reserve. The scans you took suggest some are not totally fighting fit, or perhaps will be kept to use their weapons inside the spaceport as a backup defense.`

> 			`	"To counter this, you'll need to draw them out by demanding tribute from the city. They'll keep sending defense fleets until they resign to their fate and pay you daily tribute. This is where we would like you to prove yourself. We won't help you fight the tribute defense fleets, but we'll fight the pirates waiting for our arrival and any other pirates that show up."`

Reviewer 3: I wasnt suggesting a workaround, since its not possible.
I was just suggesting that, since its an engine limitation, we find a reason for it lore wise or it'll be a plothole the way I see it. And isnt good gameplay.
If you prepare for help from 2 heavy warships and then arrive on the battle and find that you're alone...

We won't have anything to make them attack tribute fleets anytime soon. What we could do is change the whole mission to make you land and spawn the ships at that moment but that wouldn't be the same thing anymore, I don't think we should change that.

So maybe just say smth like "the humans will focus on defeating incoming ships so that you can take care of the defense fleet"? Or they will cover the Unfettered ships? Idk what we need to do, but we need to do something so it isnt this weird.
Like at the very least a big warning "you'll have to fight a consequential amount of warships alone"
```suggestion
			`	"To counter this, you'll need to draw them out by demanding tribute from the city. They'll keep sending defense fleets until they resign to their fate and pay you daily tribute. This is where we would like you to prove yourself. We won't help you fight the tribute defense fleets, but we'll fight the pirates waiting for our arrival and any other pirates that show up. Given the consequential amount of warships we expect to see, I estimate the humans may have to support us in combat, which would mean you truly will have to handle the tribute fleet alone. Prepare adequately, captain."`
```
Here, this is what I mean, although you don't have to apply this exact change.

Author: It is a good approach. I will ponder the correct wording and change this soon.

> He takes you apart to warn you

Tearing Captain Last's body apart seems unnecessarily violent for a warning about combat tactics, even for a squirrel.

Reviewer 3: you made me laugh x)
anyway idk much about english in case it wasnt obvious enough, but I tried to make this as good as I could

Author: You've come close to the right text. What matters the most is presenting a tribute demand as a reasonable battle tactic, which I think you've done well. I just need to tweak the wording.

Reviewer 3: This whole thing is weird to me
Where do the thousand ships coming in (after some time) come from even? Deep missions? where they went?

Author: You reduced that problem considerably in your PR to balance the Devil-Hide battle. 

The remaining problem is the constant inflow of capital ships. They do need some because, according to the lore, there are dangerous apex predators in the Whispering Void. Probably not a massive armada, though. There's also supposed to be a shipyard, though it shouldn't be able to make new ships like Falcons, or specialized ships like Bactrians.

Reviewer 3: True, my fleet having more firebirds that they could produce there would help a bit. If there are dangerous aliens here and the Unfettered notice them thats perfect for smth I have planned for the UHai story. Could you send me the image of those wales?

Author: They can also make smaller, low-tech, Hai outfits. Hybrid human-Hai Firebird and Leviathan builds may let us have fewer ships with a similar difficulty level.

Reviewer 3: We can do that once the balancing is merged, to make it better lore wise

Author: It was a bit of work, but I think I have a good justification for the tribute demand now. Please look at it and let me know what you think.

Reviewer 4: hybrid ships are a neat idea for a future PR <3

			label kopkow
			choice
				`	"We only need to maintain the ruse a little longer."`
					goto maintain
				`	"You don't know that we've given up on that?"`
					goto giveup
				`	"Is it still working at all?"`
					goto working
					to display
						not "Hai Reveal [B12] Devil-Hide Battle: working"
			label working
			action
				set "Hai Reveal [B12] Devil-Hide Battle: working"
			`	 "Our ruse is getting preposterous. The captains at Ultima Thule already know about the Hai, but we had to tell the crews that we were dealing with a major pirate threat. We hired a few pirate gangs to fly through the area to back our story. The Shield Beetles you've just flown here have been explained away as experimental ship designs from the movie production being tested out. Still though, our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore."`
				goto kopkow
			label giveup
			`	He blinks at you, "Wait, what?"`
				goto explain
			label maintain
			`	"We've decided on an end point to this deception?"`
			label explain

> 			`	You explain how they're just trying to keep things going long enough to resolve this mission and it is like a load has been lifted off the man, though he takes it in stride fairly professionally..`

Reviewer: ```suggestion
			`	You explain how they're just trying to keep things going long enough to resolve this mission. It is like a load has been lifted off the man, although he takes it in his stride fairly professionally.`
```

			choice
				`	"It's good to hear you have the ground combat situation under control."`
					goto control
				`	"That equipment sounds ominous."`
			`	Kopkow gives you a disturbing smile that has an 'I love my work' appearance. He departs as everyone else assembles.`
				goto leave
			label control
			`	"It's more of a collaboration, really, but like I said: need-to-know basis." He nods to you and departs as everyone else assembles.`
			label leave
			`	After a large meeting about logistics, and an unsurmountable buffet, everyone disperses. General Choot'k finds you to discuss critical tactical matters.`
			`	"There's a problem with the plan," he says, "which we found after more analysis of the scans. The bay where we need to land already has ships inside, and if we try to fly in there, they'll blow us to pieces. You have lure them out and kill them all."`
			choice
				`	"How do I lure them out?"`
					goto how
				`	"Can't we blow them up on the way down?"`

> 			`	"Have you ever tried to land on a pile of blown-up space ships in the wreck of a spaceport a half kilometer underground? It ain't easy.`

Reviewer: I can't help feeling Choot'k feels out of character for saying "ain't" without having it explained to me.

Reviewer 3: yeah he shouldnt be using contractions I think, as a Hai and Unfettered on top of that

Author: Choot'k now responds to your surprise.

			`	"The pirates won't believe it if we demand the tribute, but they will believe it if you demand it. Then their defense fleets will rush out. You have to defeat them on your own to maintain the ruse. Eventually they'll give up and pay tribute. That's when we know it's safe to land."`
			choice
				`	"Sounds good. I look forward to the tribute payments."`
					goto done
				`	"Why do I have to do that myself?"`
			`	"In your culture, governments don't demand tribute from enemy planets." He shrugs. "The pirates will see it as an obvious trap if we do it. You, though? They'll see a pilot taking advantage of the situation. Just lure them out, and blow them up, while we hold off any incoming ships."`
			choice
				`	"Sounds good. I look forward to the tribute payments."`
					goto done
				`	"Why won't our fleet help me destroy the pirates coming from the planet?"`
			`	"Ah, they could, but they won't. Consider it a test. I want to see you prove yourself in battle before I open the entrance to <planet>."`
			label done

> 			`	He seems to admire your ship for a moment. "If you succeed today you shall be Hai-friend, wherever you go."`

Reviewer: I'm glad this part flows so well now. It makes so much more sense. Before, Choot'k just sounded like he was just being annoying by making you fight the tribute fleet yourself (it is still annoying...)

Reviewer 3: Not that much now that it's been balanced, I hope?

Reviewer: Well, I was able to defeat it this time. But I hope you didn't read what I wrote last night...

Author: He isn't "being annoying." You are just suffering from culture shock. If the challenge were unsurmountable, he would not have put it before you (after a balance PR).

Reviewer: I know he isn't meant to be annoying, I'm happier that it makes sense now.

		conversation
			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. Choot'k's Shield Beetle has red engine flares; the others have blue. So if he gets into trouble you should be able to tell at a distance. The baffling won't work forever, so get to it.`
			`	(NOTE: You need to dominate this planet to finish the battle. To do that, talk to the planet and demand tribute. The planet will send multiple waves of defense fleets. Demand tribute after defeating the final wave, and they'll agree. Then land to finish the mission.)`

> 			`	(WARNING: The humans and Hai you are escorting will not help you fight the defense fleets sent in response to your tribute demands, but they will fight any other pirate ships in the system.)`

Reviewer 3: ```suggestion
			`	(WARNING: The allies you are escorting will not help you fight the defense fleets sent in response to your tribute demands, but they will fight any other pirate ships in the system.)`
```
Nobody will help, mechanically.
